### DISCUSSION TITLE
希望能增加功能：下载时默认选择最高画质

### DISCUSSION THREAD

**Discussion by Twilight-9** [the1812/Bilibili-Evolved#3299] [功能建议 / Ideas]
希望能增加功能：下载时默认选择最高画质而不是上一次的选择。这样就不用每次确认下载的画质啦（如果上一次最高只有比如720p，下一次会默认选择720p）。

**Comment by the1812** [comment]
你找个 8K 的视频, 选一下 8K 画质, 以后其他视频就都是最高了
